.github/workflows/jobs-integration-test.yml

```
@@ -0,0 +1,59 @@
+name: Integration test
+on:
+  workflow_call:
+    inputs:
+      environment_name:
+        type: string
+        required: true
+        description: The name of the environment.
+      project_prefix_name:
+        type: string
+        required: true
+        description: The prefix name of the project
+    secrets:
+      AZURE_CREDENTIALS:
+        required: true
+    
+env:
+  Bot1Url: "https://byk-dev-aks-ingress.westeurope.cloudapp.azure.com/mock-bot"
+  CentOpsUrl: "https://byk-dev-aks-ingress.westeurope.cloudapp.azure.com/centops"
+  ClassifierInternalUrl: "https://mock-classifier-service.applications.svc.cluster.local"
+  DmrInternalUrl: "https://dmr-service.applications.svc.cluster.local"
+  Bot1InternalUrl: "https://mock-bot-service.applications.svc.cluster.local"
+  GenerateTestData: "false"
+  keyvault_name: ${{ inputs.project_prefix_name }}-${{ inputs.environment_name }}-kv
+    
+jobs:
+
+  IntegrationTest:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Login to Azure
+      uses: Azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Get secrets from key vault
+      uses: Azure/get-keyvault-secrets@v1
+      with: 
+        keyvault: ${{ env.keyvault_name }}
+        secrets: 'CentOpsAdminApiKey, MockBot1ApiKey'
+      id: appSecrets
+    
+    - name: Checkout Test repo
+      uses: actions/checkout@v3
+      with:
+        repository: buerokratt/Tests
+        path: Tests
+        
+    - name: Build Test repo
```

Since we are already in the "IntegrationTest` job, we could drop referencing the repo name again.

```suggestion
    - name: Build Tests
```

---

yes, this is a hangover from the CI pipeline where it is one of many, but since this is the only repo referenced, it makes sense to simplify it

---

Actually on reflection, i prefer to keep it as it makes it clear which repo it is building. I have renamed to final step where the tests are executed.